pkg/ipmi/ipmi.go

```diff
@@ -77,6 +82,34 @@ type systemInterfaceAddr struct {
 	lun      byte //nolint:unused
 }
 
+type StandardSelEventRecord struct {
+	Timestamp    uint32
+	GenId        uint16
+	EvmRev       uint8
```

Comment:
What's evm? EVM?

Reply:
It is Event message, I renamed it to `EvM` by following IPMI SPEC.